### ISSUE TITLE
have firehose create urls to the github release page

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/ecosystem#97]
Have firehose create urls to the github release page; the url could be pre-populated w/ the release tag, changelog info, ...

**Comment by jakemac53** [comment]
https://github.com/dart-lang/ecosystem/pull/98 provides a V1 of this, but I think we could use a follow up at some point:

- Only make the tag a link to create the release once it is merged to reduce mistakes
- If we do that, then we can also specify the exact commit sha for the tag, which will help eliminate race conditions in the case that another PR lands before somebody actually creates the release.
